### PR TITLE
LPS-141435 Create Sidebar component

### PR DESCRIPTION
**Pull request by jotascript**
Story: [LPS-140048](https://issues.liferay.com/browse/LPS-140048)

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/DefinitionBuilder.js [hunk 1]
```diff
@@ -10,32 +10,16 @@
...
-
-			<div className="definition-canvas">
-				<ReactFlowProvider>
-					<ReactFlow elements={[]} minZoom="0.1" onLoad={onLoad} />
-
-					<Controls showInteractive={false} />
-					<Background size={1} />
-				</ReactFlowProvider>
-			</div>
+			<CanvasMode />
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/DefinitionBuilder.js:22]: @jotascript What do you think if we rename it to `CanvasBuilder` and `XMLBuilder` or something similar? Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Review by ramonmsp — APPROVED** [review]

**Review by rafaprax — APPROVED** [review]

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by jotascript** [comment]
ci:test:relevant

**Comment by anabuchmann** [comment]
Just started reviewing :)

**Comment by anabuchmann** [comment]
Manually forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/109717

**Pull request closed without merging**